src/libdrakvuf/libdrakvuf.h

```diff
@@ -404,6 +404,28 @@ char* drakvuf_get_filename_from_handle( drakvuf_t drakvuf,
                                         drakvuf_trap_info_t* info,
                                         addr_t handle );
 
+void drakvuf_event_fd_generate(drakvuf_t drakvuf);
```

Comment:
Does this need to be public? I think it should be called automatically by the fd_add/fd_remove functions below.